hom/cogs/views.py

```diff
@@ -14,10 +15,10 @@ def __init__(self) -> None:
         custom_id="persistent_view:groups_instructions",
     )
     async def groups_instructions(
-        self, interaction: discord.Interaction, button: discord.ui.Button
+        self, interaction: discord.Interaction, _: discord.ui.Button["Support"]
```

Comment:
Can this be explained to me? Not quite sure what the ["Support"] does.

Reply:
`Button` is generic over the View it is implemented on. In this case the `Support` View.